src/components/DropDown/index.scss

```diff
@@ -29,6 +38,38 @@
       left: auto;
     }
   }
+
+  &.lightShade {
+    .toggleButton {
+      padding-right: 0;
+      font-weight: $font-weight-semibold;
+      color: $white;
+    }
+
+    .item {
+      .icon {
+        line-height: 18px;
+
+        img,
+        span[role="img"] {
+          display: inline-block;
+          vertical-align: middle;
```

Comment:
These styles should not be nested within the shade, they should always be applied to the item if .icon is there.